themes/hive/layouts/partials/banner.html

```diff
@@ -0,0 +1,39 @@
+<!---
+  Licensed to the Apache Software Foundation (ASF) under one
+  or more contributor license agreements.  See the NOTICE file
+  distributed with this work for additional information
+  regarding copyright ownership.  The ASF licenses this file
+  to you under the Apache License, Version 2.0 (the
+  "License"); you may not use this file except in compliance
+  with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing,
+  software distributed under the License is distributed on an
+  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  KIND, either express or implied.  See the License for the
+  specific language governing permissions and limitations
+  under the License. -->
+
+<banner>
+    <div id="large-header" class="large-header">
+        <canvas id="canvas"></canvas>
+        <div class="main-title">
+            <img src="{{.Site.BaseURL}}{{ .Site.Params.hiveLogo }}" width="550" height="300"  alt="Apache Software Foundation">
+            <h1 class="text-outline black-text">Apache  Hive v4</h1>
```

Comment:
Please remove v4 from the main header. This information can get obsolete and changing the main header may interfere with search engines.

Reply:
Done